manifests/plugin/tasks.pp

```diff
@@ -8,9 +8,12 @@
 #
 # $service:: Service name
 #
+# $automatic_cleanup:: Enable automatic task cleanup using a cron job
+#
 class foreman::plugin::tasks(
-  String $package = $::foreman::plugin::tasks::params::package,
-  String $service = $::foreman::plugin::tasks::params::service,
+  String  $package = $::foreman::plugin::tasks::params::package,
```

Comment:
We don't align parameters anymore. Too much changes, huge diffs for little benefit.